### ISSUE TITLE
Are there plans to compile Ubuntu 22.04 packages?

### ISSUE THREAD

**Issue by fburolo** [jstaf/onedriver#396]
The latest LTS version of Ubuntu has been released last month, but no packages for it are available yet, and the currently existing debs don't install because of dependency issues. Are there plans to extend support to the latest Ubuntu?

Thanks!

**Comment by fburolo** [comment]
I meant 24.04, but mistyped... Sorry...

**Comment by foopsss** [comment]
There should be once jstaf gets to it. If you desperately need onedriver, I guess your only choice is to build it yourself, which is quite easy and is explained in the README.

**Comment by alexmyczko** [comment]
there already is? https://packages.ubuntu.com/search?keywords=onedriver
what is the issue you're having? works for me...

**Comment by foopsss** [comment]
> there already is? https://packages.ubuntu.com/search?keywords=onedriver
> what is the issue you're having? works for me...

OP is talking about packages provided by jstaf on its own repo. There's none for the newest LTS right now, and it's probably better if you submit a PR to update the README and let him know there is a distro package.

**Comment by fburolo** [comment]
@foopsss, thanks for coming in, and clearing it up. Indeed, I was referring to jstaf's own packages. Why? Because onedriver was never present in the (deb-based) distros I was using so far, these include Ubuntu, and I didn't notice that it was finally packaged for the official Ubuntu 24.04 repos, although I'm pretty sure I've tried installing from the official repos before posting here, but I might've made a mistake, or the package was added later... IDK.
But anyway, thank to you both for the info! This solves it for me, at least for the time being, but I think the issue might still be relevant for anyone who will eventually want to use a more up-to-date package on the LTS distro. So it's probably better left open?

**Comment by foopsss** [comment]
I wouldn't close the issue before jstaf knows there's an Ubuntu package and decides whether to keep his own builds or not, but it's not a really needed issue _per se_.

**Comment by alexmyczko** [comment]
meanwhile I created a bpo for Debian, bookworm.

**Comment by fburolo** [comment]
@foopsss Yes, so far so good, the Ubuntu package seems to be working as expected.
@alexmyczko, are you also maintaining the Ubuntu package, or just the Debian one?

**Comment by alexmyczko** [comment]
@fburolo checking this page: https://packages.ubuntu.com/noble/onedriver this says
original maintainers, listing me, BUT i do not have any influence to what canonical or ubuntu does with my stuff, usually they just copy stuff from debian... so depending on your view, you can say i'm maintaining the Ubuntu package, but also i'm not maintaining the Ubuntu package.

a little bit like schroedingers plates: https://www.reddit.com/media?url=https%3A%2F%2Fpreview.redd.it%2Fdcguq4b8nlrz.jpg%3Fauto%3Dwebp%26s%3D3bbd43ea25bb6de0f2fa3b6fbbe395cd64b42ee7

**Comment by fburolo** [comment]
@alexmyczko I see. That's actually what I thought it might be... :-D Well, so far this particular "plate" is intact and usable. If anything goes wrong, the Ubuntu devs take responsibility, or at least that's what they suggest by saying: "It should generally not be necessary for users to contact the original maintainer.", and inviting to file a bug in Launchpad instead.

**Comment by lipere123** [comment]
Hello Guy.
I am willing to dedicated some time to do the upgrade to 24.04 if someon,e guide me on the build witch have a tendancy to fail even on my side.
Also, I need a good reading for the depandency.
This is a one time offer guys, because I need Onedriver or an alternative for my HPC, quickly.
I have screwed a node for the compilation, I will need to reinstall it; Once that done bye, bye Onedriver, witch is a shame.
So witch of you is with me, please ?
I need someone to get me started a bit.
Thanks in advance.
Best Regards.
Benjamin.
